CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Major Changes
 
-- This release introduces the **AWS IoT Over-the-air Update** library(Release Candidate), **backoffAlgorithm** library, and **PKCS #11** library.
```

Comment:
This seems like a misleading update as the 202012.01 release points to the RC tag..
We should be adding an update about the OTA release under the 202103.00 section..